From 76d20274ea13439bdfa277aa18a99a1ee9a61d1c Mon Sep 17 00:00:00 2001
From: Vladimir Slavik <vslavik@redhat.com>
Date: Mon, 3 Aug 2020 15:05:53 +0200
Subject: [PATCH] Create ssh user using only existing fields (#1860058)

This fixes the bug where "homedir" and other fields do not exist on
F24_SshPwData and no user is created for SSH access to installation
environment.

Resolves: rhbz#1860058

Cherry-picked from df09030f0e7c8f350ba36ce99fa1bc33e4f45b6a
---
 utils/handle-sshpw | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/utils/handle-sshpw b/utils/handle-sshpw
index 0918610be..fc4726d88 100755
--- a/utils/handle-sshpw
+++ b/utils/handle-sshpw
@@ -47,8 +47,8 @@ for ud in userdata:
             users.set_user_password(username=ud.username, password=ud.password,
                                     is_crypted=ud.isCrypted, lock=ud.lock)
     else:
-        users.create_user(username=ud.username, password=ud.password, is_crypted=ud.isCrypted, lock=ud.lock,
-                          homedir=ud.homedir, shell=ud.shell, gecos=ud.gecos, root="/")
+        users.create_user(username=ud.username, password=ud.password, is_crypted=ud.isCrypted,
+                          lock=ud.lock, root="/")
 
     if ud.sshkey:
         # Setup the account so that only the sshkey can be used
-- 
2.23.0

